sdk/lib/drivers/sound/mmixer/wave.c

```diff
@@ -114,16 +114,17 @@ MMixerGetWaveInfoByIndexAndType(
 VOID
 MMixerInitializeDataFormat(
     IN PKSDATAFORMAT_WAVEFORMATEX DataFormat,
-    LPWAVEFORMATEX WaveFormatEx)
+    LPWAVEFORMATEX WaveFormatEx,
+    DWORD cbSize)
```

Comment:
Since the function is modified, you could take the opportunity to use SAL2 annotations:
```c
MMixerInitializeDataFormat(
    _Inout_ PKSDATAFORMAT_WAVEFORMATEX DataFormat,
    _In_ LPWAVEFORMATEX WaveFormatEx,
    _In_ DWORD cbSize)
```

Reply:
Done, thanks.